src/test/java/lotto/LottoTicketTest.java

```diff
@@ -0,0 +1,14 @@
+package lotto;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoTicketTest {
+
+    @Test
+    void 로또_번호_6개_발급() {
+        LottoTicket numbers = new LottoTicket();
+        assertThat(numbers.getNumbersAsArray().length).isEqualTo(6);
```

Comment:
배열이나 콜렉션의 크기 비교 시에는 아래처럼 작성하시는걸 추천드리고 있습니다.
```suggestion
        assertThat(numbers.getNumbersAsArray()).hasSize(6);
```

Reply:
리뷰 감사합니다 ```hasSize()``` 로 변경하였습니다 😄